WebDriver: Application Guard not supported for untrusted sites (#1283)
MicrosoftDocs/edge-developer: microsoft-edge/webdriver-chromium/index.md

For more information, navigate to the [msedgedriver container on Docker Hub][DockerHubMsedgedriver].  

## Application Guard

<!-- was "Windows Defender Application Guard" renamed to "Microsoft Defender Application Guard"; are those the same thing? -->

> Microsoft Edge (Chromium) windows that use Windows Defender Application Guard (WDAG) cannot be automated or manipulated using WebDriver-related or Selenium-related technologies.

Reviewer: ```suggestion
Untrusted sites that use Microsoft Defender Application Guard (Application Guard) cannot be automated or manipulated with Microsoft Edge Driver. Application Guard launches untrusted sites in a container and the container does not expose the remote debugging port Microsoft Edge Driver needs to communicate with the site. Your enterprise administrator defines what are trusted sites, cloud resources, and internal networks. Everything not in the trusted sites list is considered untrusted. InPrivate windows and sites on the trusted sites list can still be automated with Microsoft Edge Driver. For more information about Application Guard, navigate to: 
```

Reviewer: Let's just use the "Microsoft Defender Application Guard" name from: https://docs.microsoft.com/en-us/deployedge/microsoft-edge-security-windows-defender-application-guard and I like just referring to it as Application Guard from then on.

Reviewer 2: +1

Author: Note: URL says "windows-defender", top-of-page title says "Microsoft Defender".

Even though Microsoft Edge supports IE Mode, you can't use Microsoft Edge Driver with Microsoft Edge to test sites in IE Mode.  To test sites that require Internet Explorer, use [Internet Explorer Driver][GithubSeleniumHqWikiIEDriver] with Internet Explorer.

## Application Guard

Untrusted sites that use Microsoft Defender Application Guard cannot be automated or manipulated using Microsoft Edge Driver. Application Guard launches untrusted sites in a container, and this container doesn't expose the remote debugging port that Microsoft Edge Driver needs to communicate with the site.

> Your enterprise administrator defines what are trusted sites, including cloud resources and internal networks. Sites that are not in the trusted sites list are considered untrusted. Microsoft Edge Driver can automate both InPrivate windows, and sites on the trusted sites list. 

Author: ```suggestion
Your enterprise administrator defines what are trusted sites, including cloud resources and internal networks. Sites that are not in the trusted sites list are considered untrusted. Microsoft Edge Driver can automate both InPrivate windows, and sites on the trusted sites list, including trusted sites that use Application Guard.
```
Do we need an explicit positive statement that WebDriver does work with _trusted_ sites that use Application Guard?  Maybe even start the section with that positive statement of what is supported?  @Reezaali @zoherghadyali

Reviewer 3: SGTM

> Your enterprise administrator defines what are trusted sites, including cloud resources and internal networks. Everything not in the trusted sites list is considered untrusted. InPrivate windows, and sites on the trusted sites list, can still be automated using Microsoft Edge Driver. 

Reviewer 3: ```suggestion
Your enterprise administrator defines what are trusted sites, including cloud resources and internal networks. Sites that are not in the trusted sites list are considered untrusted. Microsoft Edge Driver can automate both InPrivate windows, and sites on the trusted sites list. 
```
